experimental/tinyobj_loader_opt.h

```diff
@@ -1523,22 +1523,44 @@ bool parseObj(attrib_t *attrib, std::vector<shape_t> *shapes,
         size_t t_count = t_offsets[t];
         size_t f_count = f_offsets[t];
         size_t face_count = face_offsets[t];
-		int material_id = -1;  // -1 = default unknown material.
+        int material_id = -1;  // -1 = default unknown material.
```

Comment:
Oops, this variable is no longer used. You could remove it

Reply:
https://github.com/syoyo/tinyobjloader/commit/e060b4f4aa4f74c1e90d4c5c6efa951f8cce0ca7

Removed!